support/spring-batch/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+dependencies {
+	compileOnly(libs.spring.boot3.batch)
+    compileOnly(libs.jakarta.persistence.api)
+    compileOnly(projects.jpqlDsl)
+    compileOnly(projects.jpqlQueryModel)
+    compileOnly(projects.jpqlRender)
+
+	testImplementation(libs.test.spring.batch)
+    testImplementation(libs.jakarta.persistence.api)
+    testImplementation(projects.jpqlDsl)
+    testImplementation(projects.jpqlQueryModel)
+    testImplementation(projects.jpqlRender)
+}
+
+kotlin {
+    jvmToolchain(17)
+}
```

Comment:
For minimum support of Spring Batch, the jvm tool prefers 8.

Reply:
As i commented above, i will create new module using jvm tool 8.